test/models/pretrained.jl

```diff
@@ -0,0 +1,130 @@
+using CounterfactualExplanations
+using CounterfactualExplanations.Evaluation
+using CounterfactualExplanations.Generators
+using CounterfactualExplanations.Data
+using CounterfactualExplanations.Models
+using DataFrames
+using Flux
+using LinearAlgebra
+using MLUtils
+using Random
+init_perturbation = 2.0
+
+models = _load_pretrained_models()
+
+@testset "Pretrained models" begin
+    for (key, value) in models
+        # The name of the dataset the model was trained on
+        dataset_name = string(key)
+        @testset "$dataset_name" begin
+            for (name, M) in value[:models]
+                name = string(name)
+                @testset "$name" begin
+                    counterfactual_data = Data.data_catalogue[:vision][key]()
+                    X = counterfactual_data.X
+                    # Randomly selected factual:
+                    Random.seed!(123)
+                    x = select_factual(counterfactual_data, rand(1:size(X, 2)))
+                    # Choose target:
+                    y = predict_label(M, counterfactual_data, x)
+                    target = get_target(counterfactual_data, y[1])
+                    generator = GravitationalGenerator()
```

Comment:
Is there a reason why you chose the gravitational generator here?

Reply:
No particular reason - it just takes too much time to iterate through all generators for all models and gravitational seemed like a good representative of generators that don't require latent space and can be used for both binary and multi-class classification problems. Added a clarifying comment to the test.